### PR TITLE
docs: Change the styling of setup instructions for better readability

### PR DESCRIPTION
**Pull request by chinmaykunkikar**
## Summary
Change the styling of setup instructions for better readability

## Description
 - Use block code for commands instead of inline code. It's easier to read, copy, and paste the commands. GitHub also has the copy button for block commands.

Copy button:
image

- Add instructions to import sample data from `sample_posts.json` file using `mongoimport` command.

- **Copy** (`cp .env.sample .env`) instead of **Rename** (`mv .env.sample .env`) because git will track it as a change and you will see .env.sample file as a deleted git object. On the other hand, since we have already ignored .env* files, git won't track it and we keep the sample file untouched.

No tracked changes here:
image

### CODE REVIEW

**Comment by krishnaacharyaa** [comment]
Thank you @chinmaykunkikar for your very inputs on bettering the readme, it is much convenient now.

One thought, how about having inline comments and make it block of code ? 
Meaning now, people have to individually copy every command every step , can we do something like club all together and replace the subheading points as comments ? 

Presently
## Description
```bash
command
```

How about
```bash
#description
command
```

And kindly rename `README` as `docs` in both PR title and commit helps in maintaining the project :)

**Comment by chinmaykunkikar** [comment]
We can do it, but I think adding a description in form of a comment will defeat the purpose of having the convenience of the copy button. It'll be a bad UX.

1. Let's say user presses the copy button on this code block, it will copy the entire contents of the block.
2. User will paste that whole list of commands to the terminal.
3. While executing, if one of the commands fail, the following commands will fail as well.
4. Also, even if all commands pass, the user will still have to hit the enter key for the last command in the code block. So there's no full automation here.
5. Additionally, the comments `# comment` will also get executed unnecessarily.
6. People will now resort to manually selecting text from those code blocks to avoid that. And that will become more tedious.

But it can be done if you want to.

**Comment by chinmaykunkikar** [comment]
> And kindly rename `README` as `docs` in both PR title and commit

done

**Comment by krishnaacharyaa** [comment]
Yeah actually even I was thinking of the tradeoffs and the last pressing enter is a good point actually, yeah it would be better this way, I completely agree with you .

File: README.md [hunk 1]
```diff
@@ -15,24 +16,77 @@ At its core, this project embodies two important aims:
...
-6. Start the backend server with `npm start`
-7. Import sample posts by copying the content from
-    - `backend/data/sample_posts.json` and insert it as a document in the
-    - `wanderlust/posts` collection in your local MongoDB database.
+
+1. **Fork and Clone the Repository**
+
+```bash
+git clone https://github.com/{your-username}/wanderlust.git
+```
```
Comment (krishnaacharyaa) [inline README.md:26]: I believe this should be https://github.com/krishnaacharyaa/wanderlust.git

File: README.md [hunk 2]
```diff
@@ -15,24 +16,77 @@ At its core, this project embodies two important aims:
...
+
+```bash
+mongoimport --db wanderlust --collection posts --file ./data/sample_posts.json --jsonArray
+```
+
+6. **Configure Environment Variables**
+
+```bash
+cp .env.sample .env
+```
```
Comment (krishnaacharyaa) [inline README.md:55]: Again very handy :)

**Comment by krishnaacharyaa** [comment]
Can you kindly add documentation of how to change the commit message in a simple bullet steps, that would be really handy for the developers, and personally I feel it involves a little git learning cover, i.e changing a commet title which is already push we need to do hard pushes  if i am not wrong.

I believe we have 2 things here, editing the latest commit message and editing multiple commit messages, 

If you could add it may be in a part of contributing.md that would be really helpful :)

File: README.md (hunk 1 above)
Comment (krishnaacharyaa) [reply README.md:26]: Oh correct, my bad. No then, we'll go with this generic user-name, makes more sense :)

**Comment by krishnaacharyaa** [comment]
> It can mess up everyone's history who are working on this project and want to merge the latest commits from the upstream.
Thank you for enlightening me, yeah this is a crucial point, i nearly missed.

@chinmaykunkikar  quick question, suppose now if contributors forget to follow the naming conventions in the commits, lets say 2 commits out of 5 which is placed in the middle, what can be a fix for such scenario ?

**Review by krishnaacharyaa — APPROVED** [review]
Thank you for all the productive and time saving approaches especially the cli commands, bang on to the point !!

**Pull request merged**